nav2_pure_pursuit_controller/README.md

```diff
@@ -0,0 +1,11 @@
+# Nav2 Pure pursuit controller
+Tutorial code referenced in https://ros-planning.github.io/navigation2/
```

Comment:
navigation.ros.org, but also give the actual URL to the tutorial in question

Reply:
https://navigation.ros.org/plugin_tutorials/docs/writing_new_nav2controller_plugin.html is the URL :smile:

Reply:
Fixed